src/assets/locales/en/translation.json

```diff
@@ -442,7 +442,9 @@
     "claim_governance_token_rewards": "Claim {{governanceTokenAmount}} {{governanceTokenSymbol}} Rewards"
   },
   "staking_page": {
-    "estimated_governance_token_rewards_tooltip_label": "The estimated amount of {{governanceTokenSymbol}} you will receive as rewards. Depends on your proportion of the total {{voteGovernanceTokenSymbol}}."
+    "estimated_governance_token_rewards_tooltip_label": "The estimated amount of {{governanceTokenSymbol}} you will receive as rewards. Depends on your proportion of the total {{voteGovernanceTokenSymbol}}.",
+    "new_vote_governance_token_gained": "New {{voteGovernanceTokenSymbol}} Gained",
+    "the_increase_in_your_vote_governance_token_balance": "The increase in your {{voteGovernanceTokenSymbol}} balance"
```

Comment:
This message is shown in tooltip, so I think it would be good to add full stop and append `_tooltip_label` to message name in order to be consistent with the rest of message names.